tox.ini

```diff
@@ -9,6 +9,7 @@ deps =
     pretend
     pytest
     ./vectors
+passenv = ARCHFLAGS LDFLAGS CFLAGS INCLUDE LIB LD_LIBRARY_PATH
```

Comment:
Ok.

On Tue, May 12, 2015 at 7:45 PM, Paul Kehrer <notifications@github.com>
wrote:

> In tox.ini
> <https://github.com/pyca/cryptography/pull/1939#discussion_r30191873>:
>
> > @@ -9,6 +9,7 @@ deps =
> >      pretend
> >      pytest
> >      ./vectors
> > +passenv = ARCHFLAGS LDFLAGS CFLAGS INCLUDE LIB LD_LIBRARY_PATH
>
> We don't use it in any of our testing right now, so we probably can leave
> it out.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/1939/files#r30191873>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084